packages/shared/lib/services/connection.service.ts

```diff
@@ -446,19 +446,24 @@ class ConnectionService {
         return result;
     }
 
-    public async getOldConnections({ days, limit }: { days: number; limit: number }): Promise<NangoConnection[]> {
+    public async getOldConnections({ days, limit }: { days: number; limit?: number }): Promise<NangoConnection[]> {
```

Comment:
why optional?

Reply:
Left over from the previous iteration. Updated with https://github.com/NangoHQ/nango/pull/2105/commits/8596099ad95ab9925fc7e81af0b9731d99741475